packages/ui-components/src/components/range/range.tsx

```diff
@@ -71,8 +73,12 @@ export class KvRange implements IRange, IRangeEvents {
 						</span>
 					)}
 					<div class="range-min-max">
-						<span>{this.min}</span>
-						<span>{this.max}</span>
+						<span>
+							{this.min} {this.unit}
```

Comment:
should we create a new property, like `minLabel`? With this we can pass whatever we want. By default these labels should assing the min and max? What you think?